libsolidity/formal/Predicate.cpp

```diff
@@ -187,12 +189,21 @@ string Predicate::formatSummaryCall(vector<smtutil::Expression> const& _args) co
 {
 	solAssert(isSummary(), "");
 
-	if (auto contract = programContract())
-		return contract->name() + ".constructor()";
-
 	if (auto funCall = programFunctionCall())
 		return funCall->location().text();
 
+	auto txVars = readTxVars(_args.at(4));
+	string value;
+	if (auto v = txVars.at("msg.value"))
+	{
+		bigint x(*v);
+		if (x > 0)
+			value = "{ msg.value: " + *v + " }";
```

Comment:
If this mimic the call syntax, then there we only use `value` and not `msg.value`.